tests/conftest.py

```diff
@@ -526,11 +526,17 @@ def s3_storage_options():
 
 
 @pytest.fixture(scope="session")
-def s3():
-    return s3fs.S3FileSystem(
-        key=os.environ.get("AWS_ACCESS_KEY_ID"),
-        secret=os.environ.get("AWS_SECRET_ACCESS_KEY"),
-    )
+def s3(s3_storage_options):
+    return s3fs.S3FileSystem(**s3_storage_options)
```

Comment:
Note this change is just cosmetic. We define `key` / `secret` above in `s3_storage_options` and then do the same thing here again. This change just reuses `s3_storage_options` instead of grabbing the environment variables again. Should be okay as both fixtures are session scoped